### ISSUE TITLE
YGGTORRENT DON'T WORK

### ISSUE THREAD

**Issue by PierreAlexx** [FlareSolverr/FlareSolverr#1291]
### Have you checked our README?

- [X] I have checked the README

### Have you followed our Troubleshooting?

- [X] I have followed your Troubleshooting

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you checked the discussions?

- [X] I have read the Discussions

### Environment

```markdown
- FlareSolverr version:3.3.21
- Last working FlareSolverr version: idk
- Operating system:windows server 2022
- Are you using Docker: no
- FlareSolverr User-Agent (see log traces or / endpoint):
2024-07-27 12:30:35 INFO     FlareSolverr 3.3.21
2024-07-27 12:30:35 INFO     Testing web browser installation...
2024-07-27 12:30:35 INFO     Platform: Windows-10-10.0.20348-SP0
2024-07-27 12:30:35 INFO     Chrome / Chromium path: C:\flaresolverr\chrome\chrome.exe
2024-07-27 12:30:35 INFO     Chrome / Chromium major version: 123
2024-07-27 12:30:35 INFO     Launching web browser...
2024-07-27 12:30:42 INFO     FlareSolverr User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
2024-07-27 12:30:42 INFO     Test successful!
2024-07-27 12:30:42 INFO     Serving on http://0.0.0.0:8191
2024-07-27 12:31:00 INFO     Incoming request => POST /v1 body: {'maxTimeout': 75000, 'cmd': 'request.get', 'url': 'https://www.ygg.re/engine/search?do=search&order=desc&sort=publish_date&category=all'}
2024-07-27 12:31:05 INFO     Challenge detected. Selector found: .lds-ring
2024-07-27 12:32:20 ERROR    Error: Error solving the challenge. Timeout after 75.0 seconds.
2024-07-27 12:32:20 INFO     Response in 80.004 s
2024-07-27 12:32:20 INFO     127.0.0.1 POST http://127.0.0.1:8191/v1 500 Internal Server Error

- Are you using a VPN: no
- Are you using a Proxy: no
- Are you using Captcha Solver: no
- URL to test this issue:
https://www.ygg.re/
```

### Description

YGGTorrent don't work

### Logged Error Messages

```text
2024-07-27 12:32:20 INFO     127.0.0.1 POST http://127.0.0.1:8191/v1 500 Internal Server Error
```

**Comment by ImLilDev** [comment]
duplicated of #1253 
no update have been pushed yet.

**Comment by ImLilDev** [comment]
duplicate of #1253
no update have been pushed yet.

**Comment by Drumkill** [comment]
I've applied the fix described in [https://github.com/FlareSolverr/FlareSolverr/issues/1253](url), it was working,
it's not working anymore... :(

**Comment by WantonOstraich** [comment]
Hi,

I have the same issue with this same website, i've put the debug log in details.

It seems it sees the iframe but cannot find the button and click it...

Thanks guys. 

<details><summary>Details</summary>
<p>

2024-07-27 19:57:59 INFO     ReqId 23086268589824 Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://www.ygg.re/engine/search?category=all&do=search&order=desc&sort=publish_date', 'proxy': {}}
2024-07-27 19:57:59 DEBUG    ReqId 23086268589824 Launching web browser...
2024-07-27 19:57:59 DEBUG    ReqId 23086268589824 Started executable: `/app/chromedriver` in a child process with pid: 1508
2024-07-27 19:58:00 DEBUG    ReqId 23086268589824 New instance of webdriver has been created to perform the request
2024-07-27 19:58:00 DEBUG    ReqId 23086266488576 Navigating to... https://www.ygg.re/engine/search?category=all&do=search&order=desc&sort=publish_date
2024-07-27 19:58:00 INFO     ReqId 23086266488576 Challenge detected. Title found: Just a moment...
2024-07-27 19:58:00 DEBUG    ReqId 23086266488576 Waiting for title (attempt 1): Just a moment...
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:01 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:03 DEBUG    ReqId 23086266488576 Waiting for title (attempt 2): Just a moment...
2024-07-27 19:58:04 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:04 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:04 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:04 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:05 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:05 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:05 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:07 DEBUG    ReqId 23086266488576 Waiting for title (attempt 3): Just a moment...
2024-07-27 19:58:08 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:08 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:08 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:08 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:09 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:09 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:09 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:11 DEBUG    ReqId 23086266488576 Waiting for title (attempt 4): Just a moment...
2024-07-27 19:58:12 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:12 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:12 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:12 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:13 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:13 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:13 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:15 DEBUG    ReqId 23086266488576 Waiting for title (attempt 5): Just a moment...
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:16 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:18 DEBUG    ReqId 23086266488576 Waiting for title (attempt 6): Just a moment...
2024-07-27 19:58:19 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:19 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:19 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:19 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:20 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:20 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:20 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:22 DEBUG    ReqId 23086266488576 Waiting for title (attempt 7): Just a moment...
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:23 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:25 DEBUG    ReqId 23086266488576 Waiting for title (attempt 8): Just a moment...
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:26 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:28 DEBUG    ReqId 23086266488576 Waiting for title (attempt 9): Just a moment...
2024-07-27 19:58:29 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:29 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:29 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:29 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:30 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:30 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:30 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:32 DEBUG    ReqId 23086266488576 Waiting for title (attempt 10): Just a moment...
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:33 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:35 DEBUG    ReqId 23086266488576 Waiting for title (attempt 11): Just a moment...
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:36 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:38 DEBUG    ReqId 23086266488576 Waiting for title (attempt 12): Just a moment...
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:39 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:41 DEBUG    ReqId 23086266488576 Waiting for title (attempt 13): Just a moment...
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:42 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:44 DEBUG    ReqId 23086266488576 Waiting for title (attempt 14): Just a moment...
2024-07-27 19:58:45 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:45 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:45 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:45 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:45 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:46 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:48 DEBUG    ReqId 23086266488576 Waiting for title (attempt 15): Just a moment...
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:49 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:51 DEBUG    ReqId 23086266488576 Waiting for title (attempt 16): Just a moment...
2024-07-27 19:58:52 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:52 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:52 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:52 DEBUG    ReqId 23086266488576 iframe found
2024-07-27 19:58:53 DEBUG    ReqId 23086266488576 Attempted to click on iframe body
2024-07-27 19:58:53 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:53 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:55 DEBUG    ReqId 23086266488576 Waiting for title (attempt 17): Just a moment...
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:56 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:58:58 DEBUG    ReqId 23086266488576 Waiting for title (attempt 18): Just a moment...
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 Timeout waiting for selector
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 Try to find the Cloudflare verify checkbox...
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 Getting ShadowRoot by selector: div:not(:has(div))
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 Cloudflare verify checkbox not found on the page. JavascriptException()
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 Try to find the Cloudflare 'Verify you are human' button...
2024-07-27 19:58:59 DEBUG    ReqId 23086266488576 The Cloudflare 'Verify you are human' button not found on the page.
2024-07-27 19:59:00 DEBUG    ReqId 23086268589824 A used instance of webdriver has been destroyed
2024-07-27 19:59:00 ERROR    ReqId 23086268589824 Error: Error solving the challenge. Timeout after 60.0 seconds.
2024-07-27 19:59:00 DEBUG    ReqId 23086268589824 Response => POST /v1 body: {'status': 'error', 'message': 'Error: Error solving the challenge. Timeout after 60.0 seconds.', 'startTimestamp': 1722092279421, 'endTimestamp': 1722092340299, 'version': '3.3.21'}
2024-07-27 19:59:00 INFO     ReqId 23086268589824 Response in 60.878 s
2024-07-27 19:59:00 INFO     ReqId 23086268589824 172.18.0.12 POST http://192.168.2.39:8191/v1 500 Internal Server Error

</p>
</details>

**Comment by darktemplar242** [comment]
Was working with fix from #1253 but stopped working after a while, Exact same issue but now bound without a fix. 

Thanks for the hard work

**Comment by billouetaudrey** [comment]
> Was working with fix from #1253 but stopped working after a while, Exact same issue but now bound without a fix.
> 
> Thanks for the hard work
> 
> Edit: Running the official Docker on Unraid if that can help

Confirm it :'(
